.travis.yml

```diff
@@ -39,8 +39,6 @@ matrix:
       env: STABILITY=dev SYMFONY_VERSION=4.0.*
     - php: 7.1
       env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=3.3.*
```

Comment:
i think we should keep this, but without stability dev. i think the stability dev should go for all of the matrix entries, actually.

Reply:
My bad. Symfony 3.3 will get security bug fixes until July.